README.md

```diff
@@ -160,10 +162,16 @@ mvn test
 - Ask us on [stackask.com](https://stackask.com)
 - Email us at [Dzung](mailto:itprono3@gmail.com)
 
-# Help us by making a donation.
+# Support Us: Make a Meaningful Donation.
 
 Currently, our operating budget is fully supported by our own salaries, and all product development is still based on voluntary contributions from a few organization members. The low budget is causing significant difficulties for us. Therefore, with a clear roadmap and an ambitious goal to provide intellectual products for the community, we would greatly appreciate your support in the form of a donation to help us take further steps. Thank you in advance for your meaningful contributions!
 
+Your donation will enable us to:
+- Expand Development
+- Enhance Support
+- Community Growth
+- Promote Openness
```

Comment:
Think these points are not headings, so I would prefer NOT using Title case here